### ISSUE TITLE
[DOCS] Repository is missing contributor docs

### ISSUE THREAD

**Issue by usrbinkat** [kubevirt/containerdisks#7]
Problem: goals, non-goals, image publishing minimum standardized requirements, and outcomes are un-disclosed blocking community contribution confidence

Proposal:
  1. Establish documentation describing repository purpose and over arching outcome objectives
  2. Establish criteria for new image contribution acceptance
  3. Document contributor guidelines

**Comment by rmohr** [comment]
@usrbinkat here a first draft on what I would see as required to onboard containerdisks. I also try to outline some future development there: https://github.com/kubevirt/containerdisks/pull/8/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5

Happy to get your feedback.

**Comment by usrbinkat** [comment]
@rmohr you guys have accelerated down that path insanely fast. bravo. My golang skills arent up to par yet. Once that is merged in I'll give it a run and then PR docs enhancement where I can.

**Comment by rmohr** [comment]
> @rmohr you guys have accelerated down that path insanely fast. bravo. My golang skills arent up to par yet. Once that is merged in I'll give it a run and then PR docs enhancement where I can.

PR is in. Looking forward to your work :)
